controllers/chat/group/groupAddMembers.js

```diff
@@ -69,11 +69,11 @@ const groupAddMembers = async (req, res) => {
           }
         );
 
-      responseChannel.channel.better_channel_member = betterChannelMember;
-      responseChannel.channel.name = newChannelName;
+      updatedChannel.channel.better_channel_member = betterChannelMember;
+      updatedChannel.channel.name = newChannelName;
 
       return ResponseSuccess(res, 'New members have been added to the group', 200, {
-        ...responseChannel,
+        ...updatedChannel,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a redundancy in the response object. The `updatedChannel` already contains the `better_channel_member` and `name` properties, so there's no need to add them again separately. This can lead to confusion and potential bugs if the values of these properties are not consistent. Here is how you can simplify it:

```diff
-      updatedChannel.channel.better_channel_member = betterChannelMember;
-      updatedChannel.channel.name = newChannelName;
-  
-       return ResponseSuccess(res, 'New members have been added to the group', 200, {
-        ...updatedChannel,
-        better_channel_member: betterChannelMember,
-        better_channel_member_object: betterChannelMemberObject,
-        name: newChannelName
+      return ResponseSuccess(res, 'New members have been added to the group', 200, updatedChannel);
```
This way, you're only returning the `updatedChannel` object which should already contain all the necessary information.


<!-- This is an auto-generated comment by OSS CodeRabbit -->